cspell.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": ["neighbour", "neighbours", "pathing"]
```

Comment:
**question**: Whats this file for?

Reply:
Otherwise VS Code complains about spelling errors. `words` is a list of words to ignore the spelling for.